### PR TITLE
Rented Book Details

### PR DESCRIPTION
**Pull request by gmazzei**
## Summary ##

**Trello card**: https://trello.com/c/qbntoEOR

This PR adds a page that shows the details of a rented book. It is opened when the user taps a cell of the table (Rentals tab).

### iPhone 5c/5/5s

simulator screen shot - iphone 5s - 2018-08-29 at 11 50 52

### iPhone 6/7

simulator screen shot - iphone 7 - 2018-08-29 at 11 51 30

### iPhone 6+/7+

simulator screen shot - iphone 7 plus - 2018-08-29 at 11 48 24

## Screencast ##

rentedbookdetail

## Known issues ##

No issues to report.

### CODE REVIEW

File: WBooks/Book/BookDetailView.swift [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        commonInit()
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        commonInit()
+    }
+    
+    private func commonInit() {
+        Bundle.main.loadNibNamed("BookDetailView", owner: self, options: nil)
```
Comment (aducret) [inline WBooks/Book/BookDetailView.swift:31]: I think it is not necessary

File: WBooks/Book/BookViewController.swift [hunk 2]
```diff
@@ -14,13 +14,16 @@ import WolmoCore
 
 class BookViewController: UIViewController {
     
-    private let bookView: BookView = BookView.loadFromNib()!
-    public let book: Book
+    internal let bookView: BookView = BookView.loadFromNib()!
     private let viewModel: BookViewModel
+    public let book: Book
```
Comment (aducret) [inline WBooks/Book/BookViewController.swift]: This book should be on the view model

File: WBooks/SubViews/Details/DetailsView.swift [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        loadDetailsView()
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        loadDetailsView()
+    }
+    
+    private func loadDetailsView() {
+        Bundle.main.loadNibNamed("DetailsView", owner: self, options: nil)
```
Comment (aducret) [inline WBooks/SubViews/Details/DetailsView.swift:36]: Check if this is needed.

**Review by aducret — CHANGES_REQUESTED** [review]

File: WBooks/Book/BookViewController.swift (hunk 2 above)
Comment (gmazzei) [reply WBooks/Book/BookViewController.swift]: Done.

File: WBooks/Book/BookDetailView.swift (hunk 1 above)
Comment (gmazzei) [reply WBooks/Book/BookDetailView.swift:31]: I tried to remove it but it is necessary for the correct binding between files. This is the only case I've found in which it's mandatory to do this. 
I followed this tutorial: https://medium.com/@brianclouser/swift-3-creating-a-custom-view-from-a-xib-ecdfe5b3a960

File: WBooks/SubViews/Details/DetailsView.swift (hunk 3 above)
Comment (gmazzei) [reply WBooks/SubViews/Details/DetailsView.swift:36]: The same as above

**Review by aducret — APPROVED** [review]

**Pull request merged**